packages/core/contracts/financial-templates/long-short-pair/LongShortPairCreator.sol

```diff
@@ -123,15 +127,15 @@ contract LongShortPairCreator is Testable, Lockable {
         shortToken.addBurner(lspAddress);
         shortToken.resetOwner(lspAddress);
 
-        emit CreatedLongShortPair(lspAddress, msg.sender);
+        emit CreatedLongShortPair(lspAddress, msg.sender, address(longToken), address(shortToken));
 
         return lspAddress;
     }
 
     // IERC20Standard.decimals() will revert if the collateral contract has not implemented the decimals() method,
     // which is possible since the method is only an OPTIONAL method in the ERC20 standard:
     // https://eips.ethereum.org/EIPS/eip-20#methods.
-    function _getSyntheticDecimals(IERC20Standard _collateralToken) public view returns (uint8 decimals) {
```

Comment:
OZ comment on decimals method:

The [`_getSyntheticDecimals` function](https://github.com/UMAprotocol/protocol/blob/b508f536ddfd94a79f93f633142bdc868b73461c/packages/core/contracts/financial-templates/long-short-pair/LongShortPairCreator.sol#L138) is described and named as a `private` function, but it is `public`. Consider making it private.